lisa/tools/git.py

```diff
@@ -172,6 +173,20 @@ def apply(
         )
         result.assert_exit_code(message=f"failed on applying patches. {result.stdout}")
 
+    def bisect(
```

Comment:
Let's make it wrapper better to use like bisect_start, bisect_stop, bisect_good, bisect_bad.

Reply:
The methods make it easy to use. The logic is here anyway. To avoid create too many methods in Git tool. You can create a bisect class, and put all methods inside.

Reply:
https://github.com/microsoft/lisa/pull/2984